ceph.spec.in

```diff
@@ -740,6 +740,7 @@ Summary:	Ceph daemon for immutable object cache
 %if 0%{?suse_version}
 Group:		System/Filesystems
 %endif
+Requires:	ceph-base = %{_epoch_prefix}%{version}-%{release}
```

Comment:
There's no mention of "ceph-base" in the commit message,  so I just want to make sure this is intentional here?

Reply:
"ceph-base" pulls in ceph-common, but maybe we don't need the full -base package here?

Reply:
Yes, I went with `ceph-base` because that is what all daemons require on the rpm side.  It looks like only `ceph-test` package pulls `ceph-common` directly, everything else pulls it transitively through `ceph-base`.

Reply:
> "ceph-base" pulls in ceph-common, but maybe we don't need the full -base package here?

Our messages crossed.  We probably don't (and not just for immutable object cache, but also for rbd-mirror, cephfs-mirror, etc), but I didn't want to deviate from what is already there in a bug fix intended for backporting.